### ISSUE TITLE
[Examen] Horarios del doodle

### ISSUE THREAD

**Issue by jvlara** [UC-IIC3253/2021#58]
Hola,
Cuando inscribí el horario para dar el examen no estaba logueado y ya no recuerdo en que hora exactamente lo puse... ¿Existe alguna manera de verlo? 😄

**Comment by fernandamansilla** [comment]
Tengo el mismo problema, se que me inscribí el sábado pero no anote la hora. Agradecería mucho si me pueden confirmar el horario :)

**Comment by frvalenzuela** [comment]
Tengo el mismo problema, también agradecería si pueden confirmar los horarios.

**Comment by fprebolledo** [comment]
Same

**Comment by Okhan97** [comment]
@marceloarenassaavedra

**Comment by frvalenzuela** [comment]
Comentó que los profesores me respondieron por correo mi horario

**Comment by marceloarenassaavedra** [comment]
Sí, los horarios fueron confirmados individualmente a las personas que los pidieron.

Saludos!
